components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/Utils.java

```diff
@@ -181,9 +181,7 @@ public static void setSOAPFault(MessageContext messageContext, String code,
         }
     }
     
-    public static String getAllowedOrigin(String currentRequestOrigin) {
-    	String allowedOrigins = ServiceReferenceHolder.getInstance().getAPIManagerConfiguration().
-    	        getFirstProperty(APIConstants.CORS_CONFIGURATION_ACCESS_CTL_ALLOW_ORIGIN);
+    public static String getAllowedOrigin(String currentRequestOrigin,String allowedOrigins) {
```

Comment:
method removed due to no usage and code moved 
https://github.com/wso2/carbon-apimgt/pull/792